src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/WindowsIdentity.cs

```diff
@@ -138,7 +138,9 @@ public WindowsIdentity(string sUserPrincipalName)
                 // the ArgumentNullException (provided that the prior LSA calls didn't fail first.) To make this compat decision explicit, we'll null check ourselves
                 // and simulate the exception from Encoding.Unicode.GetBytes().
                 if (sUserPrincipalName == null)
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
                     throw new ArgumentNullException("s");
+#pragma warning restore C2208
```

Comment:
Why not `throw new ArgumentNullException(nameof(sUserPrincipalName));`?  If it's because of the above comment, I think we can just change it and delete that comment, but I'll leave that up to @bartonjs.  (We're making a ton of such changes in this PR, and we've made lots of them in the past, so I'm not sure why this one would be special.)

Reply:
Yeah, let's delete the comment and fix the argument name.  (And then, presumably, fix the test that resulted in the giant comment :smile:)